src/main/java/fi/metatavu/famifarm/events/SowingEventController.java

```diff
@@ -113,7 +128,11 @@ public SowingEvent updateSowingEvent(SowingEvent sowingEvent, Batch batch, Offse
     sowingEventDAO.updatePotType(sowingEvent, potType, modifier);
     sowingEventDAO.updateAmount(sowingEvent, amount, modifier);
     sowingEventDAO.updateAdditionalInformation(sowingEvent, additionalInformation, modifier);
-    
+
+    int difference = amount - sowingEvent.getAmount();
```

Comment:
It would be less error prone if the new total would be calculated from data again instead of adding or removing from existing total